webapp/WEB-INF/templates/admin/user/attribute/radiobutton/html_code_form_attribute_radio_button.html

```diff
@@ -30,15 +20,6 @@
 		<#assign orientation = 'vertical'>
 	</#if>
 	
-	<#if !attribute.mandatory>

```

Comment:
This was a bogus report. When the attribute is not mandatory, we need the "no value" field because we can't unselect a radio button.